src/components/Flag/Flag.tsx

```diff
@@ -27,11 +28,24 @@ const getSourceByIsoCode = (isoCode: MuiTelInputCountry | null) => {
   return ''
 }
 
-const Flag = ({ size = 'small', isoCode, countryName = '' }: FlagProps) => {
+const defaultGetFlagSources: GetFlagSources = (isoCode, width) => {
+  return {
+    png: `https://flagcdn.com/w${width}/${isoCode}.png`,
+    webp: `https://flagcdn.com/w${width}/${isoCode}.webp`
+  }
+}
+
+const Flag = ({
+  size = 'small',
+  isoCode,
+  countryName = '',
+  getFlagSources = defaultGetFlagSources
+}: FlagProps) => {
   const isoCodeFormatted = isoCode ? isoCode.toLowerCase() : ''
   const sourceFound = getSourceByIsoCode(isoCode)
   // see https://flagpedia.net/download/api for the valid width
   const width = size === 'small' ? 40 : 80
```

Comment:
nah, 40 or 80 is not valid anymore as there is no only 1 cdn.. You should let the developer choose the `width` himself.